core/src/main/scala/com/pingcap/tispark/TiDBOptions.scala

```diff
@@ -65,7 +65,10 @@ class TiDBOptions(@transient val parameters: CaseInsensitiveMap[String]) extends
   val skipCommitSecondaryKey: Boolean =
     parameters.getOrElse(TIDB_SKIP_COMMIT_SECONDARY_KEY, "true").toBoolean
 
-  val sampleFraction: Double = parameters.getOrElse(TIDB_SAMPLE_FRACTION, "0.01").toDouble
+  val regionSplitNum: Int = parameters.getOrElse(TIDB_REGION_SPLIT_NUM, "3").toInt
```

Comment:
`TIDB_REGION_SPLIT_NUM` -> `TIDB_REGION_SPLIT_NUM_FOR_TEST`

Reply:
I rethink about this, this option can also used by user when they want to split region although data to be inserted is very small.

Reply:
pls update document about this parameter

Reply:
which one? datasource_userguide_api?

Reply:
datasource_api_userguide.md